src/main/java/Duke.java

```diff
@@ -1,10 +1,80 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
 public class Duke {
+    public static ArrayList<Task> tasks = new ArrayList<Task>();
+    public static final Scanner SCANNER = new Scanner(System.in);
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userInput = "";
+        Output.printStart();
+
+        while (true) {
+            userInput = SCANNER.nextLine();
+            Output.printBorder();
+            if (userInput.trim().equals("bye")) {
+                Output.printBye();
+                break;
+            } else if (userInput.startsWith("list")) {
+                System.out.println("Here are the tasks in your list: ");
+                for (int i = 0; i < tasks.size(); i++) {
+                    System.out.print(i + 1 + ".");
+                    System.out.println(tasks.get(i));
+                }
+            } else if (userInput.trim().startsWith("done")) {
+                int processedInput;
+                processedInput = Integer.parseInt(userInput.replaceAll("[^0-9]", "")) - 1;
```

Comment:
For wrapped lines, maybe you can consider indenting with 8 spaces more than the parent line.